doc/user/content/get-started/install-materialize.md

```diff
@@ -0,0 +1,145 @@
+---
+title: "Install Materialize locally"
+description: "Download the Docker image and run Materialize locally for evaluation"
+menu:
+  main:
+    parent: "get-started"
+    weight: 16
+    name: "Install Materialize locally"
+
+---
+
+For testing/evaluation purposes, you can download the Materialize Docker image
+and run Materilize locally in a Docker container.  The Docker image:
+
+- Includes the [Quickstart](/get-started/quickstart) data source
+
+- Supports connecting your data sources
+
+- Provides [community support via
+  Slack](https://materializecommunity.slack.com/join/shared_invite/zt-2bad5ce4i-ZsiPWI5jd7Q9pRDGYj3dkw#/shared-invite/email)
+
+To more fully evaluate
+Materialize and its features, Materialize offers a [free trial
+account](https://materialize.com/register/?utm_campaign=General&utm_source=documentation).
+
+{{< warning >}}
+
+The Docker image is for <redb> testing/evaluation purposes only. It is not
+suitable for production deployments</redb>.  See also [Limitations](#limitations).
+
+{{</ warning >}}
+
+### License and Privacy Policy
+
+- Use of the Docker image is subject to Materialize's [BSL
+  License](https://github.com/MaterializeInc/materialize/blob/main/LICENSE).
+
+- By downloading the Docker image, you are agreeing to Materialize's [privacy
+  policy](https://materialize.com/privacy-policy/).
+
+### Prerequisites
+
+- Docker. If [Docker](https://www.docker.com/) is not installed, refer to its
+[official documentation](https://docs.docker.com/get-docker/) to install.
+
+### Run Materialize in a Docker container
+
+{{< note >}}
+
+- Use of the Docker image is subject to Materialize's [BSL
+   License](https://github.com/MaterializeInc/materialize/blob/main/LICENSE).
+
+- By downloading the Docker image, you are agreeing to Materialize's [privacy
+   policy](https://materialize.com/privacy-policy/).
+
+{{</ note >}}
+
+1. In a terminal, issue the following command to run a Docker container from the
+   Materialize image. The command downloads the Materialize image, if one has
+   not been already downloaded.
+
+   ```sh
+   docker run -v mzdata:/mzdata -p 6875:6875 -p 6876:6876 materialize/materialized
+   ```
+
+   When running locally:
+
+   - Materialize logs are emitted to the `stderr`.
+   - The SQL interface is available on port `6875`.
+   - The HTTP interface is available on port `6876`.
```

Comment:
Do we need to mention the HTTP interface, since it's full-on legacy? I think it's potentially confusing + we should not give users the false impression that this is anything remotely close to the information displayed in the Materialize Console. Also, we for sure don't want to be fielding questions about it.

Reply:
Will remove. cc @ggnall  in case he wants to remove it from the downloads page as well.